README.md

```diff
@@ -7,3 +7,14 @@ Lavaplayer server benchmarker. How many lavaplayer tracks can your server handle
 ```
 java -jar lavamark-1.0.jar
 ```
+
+## Lavamark Options
+You can specify any of the following options as a command-line argument when launching Lavamark.
+
+| Option           | Description                                                                                |
+|------------------|--------------------------------------------------------------------------------------------|
+| `h`/`help`       | Displays Lavamark's available options.                                                     |
+| `i`/`identifier` | The identifier/URL of the track/playlist to use for the test.                              |
+| `b`/`block`      | The IPv6 block to use for rotation, specified as CIDR notation.                            |
+| `s`/`step`       | The number of players to spawn after a fixed interval. Be careful when using large values. |
+| `t`/`transcode`  | Simulate a load by forcing transcoding.                                                    |
```

Comment:
```suggestion
| Option              | Description                                                                                        |
|---------------------|----------------------------------------------------------------------------------------------------|
| `-h`/`--help`       | Displays Lavamark's available options.                                                             |
| `-i`/`--identifier` | The identifier or URL of the track/playlist to use for the benchmark.                              |
| `-b`/`--block`      | The IPv6 block to use for rotation, specified as CIDR notation. Only applies to YouTube currently. |
| `-s`/`--step`       | The number of players to spawn after two seconds. Be careful when using large values.              |
| `-t`/`--transcode`  | Simulate a load by forcing transcoding.                                                            |
```

Additionally it would also be nice if these descriptions were the same as the ones defined in the code.